website/src/react-native/versions.js

```diff
@@ -34,20 +45,29 @@ var versions = React.createClass({
         {
           title: 'current',
           path: '/react-native',
+          release: null
         },
       ].concat(versions);
     }
 
-    var versionsLi = versions.map((version) =>
-      <li><a href={version.path}>{version.title}</a></li>
-    );
     return (
       <Site section="versions" title="Documentation archive">
-        <section className="content wrap versions documentationContent">
-          <h1>Documentation archive</h1>
-          <ul>
-            {versionsLi}
-          </ul>
+        <section className="content wrap documentationContent nosidebar">
+          <div className="inner-content">
+            <h1>React Native Versions</h1>
+            <p>React Native is following a 2-week train release. Every two weeks, a Release Candidate (rc) branch is created off of master and the previous rc branch is being officially released.</p>
```

Comment:
Just a v1 of the text, I'd like to ship it as is and then people can submit more pull requests to make it good.